alerts_consumer_test.go

```diff
@@ -5,11 +5,14 @@ import (
 	"testing"
 	"time"
 
-	"golang.org/x/net/context"
+	"github.com/aws/aws-sdk-go/service/cloudwatch/cloudwatchiface"
 
+	"github.com/Clever/aws-sdk-go-counter/aws"
```

Comment:
```suggestion
	"github.com/aws/aws-sdk-go/aws"
```
also if you `git pull` the latest of `github.com/Clever/aws-sdk-go-counter` on your computer it should no longer have this package